src/cljs/main/broadfcui/utils.cljs

```diff
@@ -245,6 +245,9 @@
 (defn first-matching-index [pred coll]
   (first (keep-indexed (fn [i x] (when (pred x) i)) coll)))
 
+(defn vec-contains? [coll item]
+  (some? (some #(= item %) coll)))
```

Comment:
I suggest `(contains? (set coll) item)`. It also works on any sequence, so I would suggest the name `seq-contains?`